packages/web-service/src/pages/eligibility/eligible.njk

```diff
@@ -4,7 +4,6 @@
 {% block pageContent %}
 
     {% if credentials %}
```

Comment:
It might be neater to use `{% if not credentials %}` and put the content in that block rather than having an empty `{% if credentials %}` block.